### PR TITLE
Fix Primer token references and details

### PR DESCRIPTION
**Pull request by lecoqlibre**
As discussed with @elf-pavlik, some fixes and improvements in the Primer:
- Steps 7 and 12 of the Request flow diagram was refering to access token instead of ID token.
- Show the kid in the ID Token at step 19 of the Authorization code PKCE flow.
- Add a reference to RFC7515 section 4.1.4 in step 12 of the Request flow.

See https://github.com/solid/solid-oidc/issues/229.

### CODE REVIEW

**Review by elf-pavlik — APPROVED** [review]
LGTM

**Review by woutermont — APPROVED** [review]

File: primer/index.bs [hunk 1]
```diff
@@ -915,6 +916,8 @@ Notice that the `keys` field is an array, so an OP could have multiple public ke
 Using the `kid` value in the ID token, the AS searches the OP public keys for one that matches the
 one used to sign the ID token. If no public key is found, the AS must reject the request.
 
+Details about the `kid` can be found in the [RFC7515 section 4.1.4](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.4).
```
Comment (acoburn) [inline primer/index.bs:919]: Ordinarily, with Bikeshed, you would refer to an RFC like so:

`[[RFC7515#section-4.1.4]]`

(For normative references, you would include an exclamation point `[[!RFC7515]]`, but for the primer, everything should be informative).

**Review by acoburn — COMMENTED** [review]
Generally, this is good. One minor comment about the formatting of an informative RFC link.

File: primer/index.bs (hunk 1 above)
Comment (lecoqlibre) [reply primer/index.bs]: OK thanks @acoburn. I just changed it in the [last commit](https://github.com/solid/solid-oidc/pull/230/commits/deb20ba003b7685058e116f005a97a2d1fcd0f3c).

**Pull request merged**